promptsource/templates/xnli/ar/templates.yaml

```diff
@@ -42,33 +48,56 @@ templates:
   8bda8e36-c881-4c9a-a3a9-eec68388a6f6: !Template
     answer_choices: نعم ||| ربما ||| لا
     id: 8bda8e36-c881-4c9a-a3a9-eec68388a6f6
-    jinja:"{{premise}}
-     هل يمكننا استنتاج 
-     "{{hypothesis}}"
-      نعم، لا أو ربما؟ ||| {{ answer_choices[label] }} "
+    jinja: '{{premise}} هل يمكن القول بالفرضية التالية  "{{hypothesis}}" بناء على
+      ما سبق؟ أجب بنعم أو ﻻ أو ربما ؟ ||| {{ answer_choices[label] }} '
     metadata: !TemplateMetadata
       choices_in_prompt: true
       languages:
-      - en
+      - ar
       metrics:
       - Accuracy
       original_task: true
     name: justified in saying
     reference: Webson & Pavlick 2021
+  97d2f061-06b0-4aa3-af53-871a2b06748f: !Template
+    answer_choices: صح ||| غير ذلك ||| خطأ
+    id: 97d2f061-06b0-4aa3-af53-871a2b06748f
+    jinja: '{{premise}} سؤال: {{hypothesis}} صح ، خطأ إو غير ذلك ||| {{ answer_choices[label]
+      }}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: true
+      languages:
+      - ar
+      metrics:
+      - Accuracy
+      original_task: true
+    name: GPT-3 style
+    reference: 'Same as reported in Figure G7 of the GPT-3 paper, except that there
+      is no task identifying tokens like "anli R1: ".'
+  ba194be7-4818-4e24-b5f9-bbcccaa4e166: !Template
+    answer_choices: نعم ||| ربما ||| لا
+    id: ba194be7-4818-4e24-b5f9-bbcccaa4e166
+    jinja: 'بناء على المقدمة التالية {{premise}} هل يمكن أن تليها الفرضة التالية:
+      {{hypothesis}} أجب بـ نعم أو ربما أو لا ؟  ||| {{ answer_choices[label] }}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: true
+      languages:
+      - ar
+      metrics:
+      - Accuracy
+      original_task: true
+    name: does it follow that
+    reference: Sanh et al. 2021
   e44c2947-f8e0-49ea-9770-e59f0581a921: !Template
     answer_choices: أكيدة ||| ممكنة ||| مستحيلة
     id: e44c2947-f8e0-49ea-9770-e59f0581a921
-    jinja: "افترض أن 
-    {{premise}} 
-    لذلك فإن  
-    \"{{hypothesis}}\" 
-    تعتبر
-      {{\"أكيدة\"}}, {{\"ممكنة\"}}, or {{\"مستحيلة\"}}? 
-      ||| {{ answer_choices[label] }}"
+    jinja: "اذا كانت لدينا المقدمة الصحيحة التالية {{premise}} \nفإن, \"{{hypothesis}}\"\
+      \ هي دعوى {{\"مؤكدة\"}} أو {{\"ممكنة\"}} أو {{\"مستحيلة\"}}؟ ||| {{ answer_choices[label]\
```

Comment:
أكيدة

Reply:
Hi @zaidalyafeai, I added "مؤكدة" to the answer choices because I think "مؤكدة" is more aligned with the context. However, I am not sure. if you have any suggestions to make it better please let me know. Thanks for all your comments. It is really helpful.

Reply:
Sounds good